abpframework/abp
# Fix Mvc dynamic Form DateTimePicker Error

Resolves #16494 

![image](https://user-images.githubusercontent.com/58659931/236678726-fc5d5d2f-c830-4962-857d-c869d1b365f9.png)


### Checklist


### How to test it?

Create a modal and have abp-date-picker in the modal. The datepicker should work when you open the modal
